docs/source/changelog.rst

```diff
@@ -28,10 +28,11 @@ Changelog
         * Add development install instructions to contributing.md (:pr:`1030`)
     * Testing Changes
         * Add ``required`` flag to CircleCI codecov upload command (:pr:`1035`)
+        * Add fixture for ``ft.demo.load_mock_customer`` (:pr:`1036`)
```

Comment:
This item should be added to the `Future Release` section above. Be sure to uncomment that section by removing the two dots `..` before the `Future Release` test.